src/Tribe/Shortcode/Shortcode_Abstract.php

```diff
@@ -63,6 +49,55 @@ abstract class Shortcode_Abstract implements Shortcode_Interface {
 	 */
 	protected $content;
 
+	/**
+	 * Get the shortcode tag, forced to lowercase with underscores.
+	 *
+	 * If `$this->tag` exists, use it, else it will be created dynamically from this class' name.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_tag() {
+		if (
+			! empty( $this->tag )
```

Comment:
Should we define the protected tag property?

Reply:
If you want to use the prop, you'd add it in the child class, not necessary in abstract class.
*Should we???* Personally, no because we'd like the shortcode class to be descriptively named and therefore the shortcode tag 99% of the time should be generated. Defining the prop in the abstract would encourage its use more...